### DISCUSSION TITLE
Restore by PVC name

### DISCUSSION THREAD

**Discussion by orellazri** [k8up-io/k8up#1024] [Q&A]
Is it possible to restore a PVC by its name (using the latest snapshot), instead of by snapshot ID?
I'm thinking of a worst-case scenario where a cluster needs to start up from scratch and doesn't contain the Snapshot CRs, it would be a pain to go through every Restic repository with the restic cli to get the snapshots IDs.

**Comment by s4ndalHat** [comment]
Looking for this feature too if its possible.
